src/payment/strategies/masterpass/masterpass-payment-initialize-options.ts

```diff
@@ -0,0 +1,11 @@
+export interface MasterpassPaymentInitializeOptions {
+    /**
+     * The gateway where the masterpass payment will be processed.
+     */
+    gateway?: string;
```

Comment:
You shouldn't need to pass the `gateway` field again, because it should already be passed when you call `initialize`. Could you please remove?